src/ring_theory/ideal/operations.lean

```diff
@@ -1604,3 +1611,86 @@ begin
 end
 
 end ring_hom
+
+namespace double_quot
+open ideal
+variables {R : Type u} [comm_ring R] (I J : ideal R)
+
+/-- define `quot_left_to_quot_sum` to be the obvious ring hom `R/I → R/(I ⊔ J)` -/
+def quot_left_to_quot_sum : I.quotient →+* (I ⊔ J).quotient :=
+  ideal.quotient.inclusion I (I ⊔ J) (le_sup_left)
+
+/-- This will be used to lift `quot_left_to_quot_sum` to a map `(R/I)/J' → R/(I ⊔ J)`-/
+lemma img_left_in_ker (x : I.quotient) (hx : x ∈ J.map(ideal.quotient.mk I)) :
+  quot_left_to_quot_sum I J x = 0 :=
+begin
```

Comment:
I think this is a better statement
```suggestion
/-- This will be used to lift `quot_left_to_quot_sum` to a map `(R/I)/J' → R/(I ⊔ J)`-/
lemma map_mk_le_ker_quot_left_to_quot_sum :
  J.map (ideal.quotient.mk I) ≤ (quot_left_to_quot_sum I J).ker :=
begin
  intros x hx,
```
The best bit is that you don't have to change your proof!